speech-commands/demo/index.js

```diff
@@ -324,3 +379,99 @@ startTransferLearnButton.addEventListener('click', async () => {
   startTransferLearnButton.textContent = 'Transfer learning complete.';
   startButton.disabled = false;
 });
+
+downloadFilesButton.addEventListener('click', () => {
+  const basename = getDatasetFileBasename();
+  const artifacts = transferRecognizer.serializeExamples();
+
+  // Trigger downloading of the manifest .json file.
+  const manifestAnchor = document.createElement('a');
+  manifestAnchor.download = `${basename}.json`;
```

Comment:
optional: i like the debuggability of the separate JSON file, but might it be more user-friendly to package into one file instead of two? You could either include the JSON at the front of the bin file, or (bigger) maybe base64 encode the spectrograms into the JSON?